packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1333,7 +1333,7 @@ class SelectionOverlay {
 
   /// A pair of handles. If this is non-null, there are always 2, though the
   /// second is hidden when the selection is collapsed.
-  List<OverlayEntry>? _handles;
+  (OverlayEntry, OverlayEntry)? _handles;
```

Comment:
For clarity, could we maybe use named record fields i.e. `(start: OverlayEntry, end: OverlayEntry)` ?

Reply:
done